src/lib.rs

```diff
@@ -396,85 +396,43 @@ pub fn write_global(store: &mut Store, globaladdr: GlobalAddr, val: values::Valu
 
 /// Instantiate a module
 pub fn instantiate_module(store: &mut Store, module: ast::Module, extern_vals: &[ExternVal]) -> Result<Rc<ModuleInst>, Error> {
-
-	// 1. If module is not valid, then: Fail
+	// fail if module is invalid
 	if !valid::is_valid(&module) {
 		return Err(Error::InvalidModule)
 	}
 
-	// 2. Assert: module is valid with external types externtype^{m}_{im} classifying its import
-	// 3. If the number m of imports is not equal to the number n of provided external values, then: Fail
+	// ensure that the number of provided exports matches the number of imports
```

Comment:
The comments were the ones from [the spec](https://webassembly.github.io/spec/core/exec/modules.html#instantiation).

Reply:
I noticed that, but most of them were
1. outdated since the spec has changed
2. cryptic and not helping (e.g., `externtype^{m}_{im}`)
3. unsued (e.g., `2. Assert:...`)